mmv1/products/healthcare/terraform.yaml

```diff
@@ -132,6 +132,12 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     properties:
       creationTime: !ruby/object:Overrides::Terraform::PropertyOverride
         exclude: true
+      parserConfig: !ruby/object:Overrides::Terraform::PropertyOverride
+        update_mask_fields:
+          - "parser_config.allow_null_header"
```

Comment:
Will this change also makes other field immutable? eg. `labels`...

Reply:
No, this is specific to the `parser_config` block, and it's basically saying "if there is a change to `parser_config`, send these in the `updateMask` specifically". Whereas before we were only sending `parserConfig` to the `updateMask`, which in turn assumed we were trying to update `parserConfig.version` (as a part of the block), which we were not.